site2/docs/io-jdbc-sink.md

```diff
@@ -25,6 +26,8 @@ The configuration of all JDBC sink connectors has the following properties.
 | `key`       | String | false    | " " (empty string) | A comma-separated list containing the fields used in `where` condition of updating and deleting events.                  |
 | `timeoutMs` | int    | false    | 500                | The JDBC operation timeout in milliseconds.                                                                              |
 | `batchSize` | int    | false    | 200                | The batch size of updates made to the database.                                                                          |
+| `insertMode` | enum( INSERT,UPSERT,UPDATE) | false    | INSERT | If it is configured as UPSERT, the sink will use upsert semantics rather than plain INSERT/UPDATE statements. Upsert semantics refer to atomically adding a new row or updating the existing row if there is a primary key constraint violation, which provides idempotence. |
```

Comment:
```suggestion
| `insertMode` | enum( INSERT,UPSERT,UPDATE) | false    | INSERT | If it is configured as UPSERT, the sink uses upsert semantics rather than plain INSERT/UPDATE statements. Upsert semantics refer to atomically adding a new row or updating the existing row if there is a primary key constraint violation, which provides idempotence. |
```
Write in the simple present tense as much as possible if you are covering facts that were, are, and forever shall be true.https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.e8uqh1awkcnp